dts/x86/quark_se_c1000_devboard.fixup

```diff
@@ -0,0 +1,9 @@
+#define CONFIG_UART_QMSI_0_BAUDRATE	INTEL_QMSI_UART_B0002000_CURRENT_SPEED
+#define CONFIG_UART_QMSI_0_NAME		INTEL_QMSI_UART_B0002000_LABEL
```

Comment:
It is generated, the fixup file is meant to map from the generated defines to more "generic" ones that drivers can use.  Eventually the fixup files should go away when we can generate the code/structs directly.